docs/source/analyst/flowclient/example_usage.ipynb

```diff
@@ -195,19 +195,19 @@
    },
    "outputs": [],
    "source": [
-    "dl1 = flowclient.daily_location(\n",
+    "dl1 = flowclient.daily_location_spec(\n",
     "    date=\"2016-01-01\",\n",
     "    aggregation_unit=\"admin3\",\n",
     "    method=\"last\",\n",
     "    subscriber_subset=None,\n",
     ")\n",
-    "dl2 = flowclient.daily_location(\n",
+    "dl2 = flowclient.daily_location_spec(\n",
```

Comment:
I know we're planning an overhaul of these, but let's open an issue to track that?